_board/CC03.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: download
+board_id: "cc03"
```

Comment:
needs to match https://github.com/adafruit/circuitpython/tree/master/ports/atmel-samd/boards/xinabox_cc03